app/(auth)/(tabs)/transactions.tsx

```diff
@@ -1,102 +1,308 @@
+        <ThemedText style={styles.subHeading2Text}>Total # Collected</ThemedText>
+            <ThemedView style={[styles.transactionContainer, {backgroundColor: theme.colors.card}]}>
+              <Image
+                source={require('@/assets/images/1Point_Logo.png')}
+                style={styles.pointAmounts}
+              />
+              <ThemedText style={[styles.transactionPreviewText, {color: theme.colors.text}]}>120</ThemedText>
+            </ThemedView>
+            <ThemedView style={styles.backLabelContainerUp}>
+                  <ThemedText style={styles.backLabel}>+8% - Last Month</ThemedText>
+                </ThemedView>
+          </ThemedView>
+
+
+        </ThemedView>
+
+        <ThemedView style={{backgroundColor: theme.colors.background}}>
+        <ThemedView style={[styles.row, styles.shadowProp, {backgroundColor: theme.colors.card}]}>
+        <ThemedText style={styles.subHeading2Text}>    Total # Issued  </ThemedText>
+            <ThemedView style={[styles.transactionContainer, {backgroundColor: theme.colors.card}]}>
+              <Image
+                source={require('@/assets/images/1Point_Logo.png')}
+                style={styles.pointAmounts}
+              />
+              <ThemedText style={[styles.transactionPreviewText, {color: theme.colors.text}]}>178</ThemedText>
+            </ThemedView>
+            <ThemedView style={styles.backLabelContainerUp}>
+                  <ThemedText style={styles.backLabel}>+2% - Last Month</ThemedText>
+                </ThemedView>
+          </ThemedView>
+
 
-export default function TabTwoScreen() {
+        </ThemedView>
+          
+  
+        </ThemedView>
+      );
+    };
+
+    const mapTransactions = () => {
+      return (
+        <ThemedView>
+          {transactionArray.map((transaction) => {
+            return (
+              <TransactionRow transactionAmount={transaction.transactionAmount} transactionLocation={transaction.transactionLocation} transactionDate={transaction.transactionDate} transactionCustomerId={transaction.transactionCustomerId} transactionStatus={transaction.transactionStatus}/>
+            );
+          })}
+        </ThemedView>
+      );
+    }
+
+  // Render the transactions section
+  const renderTransactions = () => {
+    return (
+      <ThemedView style={{paddingBottom: 120, backgroundColor: theme.colors.background}}>
+       {mapTransactions()}
+      </ThemedView>
+    );
+  };
+
+  // Render the home screen 
   return (
-    <ParallaxScrollView
-      headerBackgroundColor={{ light: '#D0D0D0', dark: '#353636' }}
-      headerImage={<Ionicons size={310} name="code-slash" style={styles.headerImage} />}>
-      <ThemedView style={styles.titleContainer}>
-        <ThemedText type="title">Transaction</ThemedText>
+    <SafeAreaView style={[styles.main, {backgroundColor: theme.colors.background}]}>
+      <ScrollView style={[styles.mainContainer, {backgroundColor: theme.colors.background}]}>
```

Comment:
We should have a 'sticky' header to indicate what page you are on? i.e. having the scrollview only in the view where we render the transactions, right now it looks odd to have the header scroll out of view 

https://github.com/user-attachments/assets/7b271f2e-996a-45ab-88ff-394812e59c7e

Reply:
Weirdly enough, the sticky header is what I'm working on right now. Right before your comment.